packages/components/src/button/src/Button.css

```diff
@@ -351,12 +351,12 @@ a.o-ui-button {
 .o-ui-button-inherit-style.o-ui-button-primary .o-ui-button-end-icon {
     fill: var(--o-ui-white);
     color: var(--o-ui-white);
-    opacity: .9;
+    opacity: 0.9;
```

Comment:
@fraincs Any opinion on this? at the moment, we have both 0.X and .X. Should we settle on one. Do you prefer with or without the 0?

Reply:
No strong opinion, although in the case of opacity it has no use, as the maximum is 1.0